examples/gradient.py

```diff
@@ -1,8 +1,8 @@
 import blend2d
 import numpy as np
-from skimage.io import imsave
+import matplotlib.pyplot as plt
```

Comment:
Happy to consider this, but it's out of scope for the proposed change here.

Reply:
Well matplotlib allow me to visualise the picture automatically. However I agree with you it is totally out of scope.